src/pheval/utils/constants.py

```diff
@@ -0,0 +1,4 @@
+PHEVAL_GENE_RESULTS_DIR = "pheval_gene_results/"
+PHEVAL_VARIANT_RESULTS_DIR = "pheval_variant_results/"
+TOOL_RESULTS_DIR = "results/"
+RUNNER_INPUT_COMMANDS_DIR = "runner_input_commands/"
```

Comment:
remind me why this is here? What was this again?

Reply:
No I meant, what is this DIR: `RUNNER_INPUT_COMMANDS_DIR`?

Reply:
This is the tool input commands directory, i.e. for Exomiser, depending on whether you want to run the jobs in batches or all in one go, the directory will contain the file(s) (which contains the commands) for exomiser to run the jobs. 

`--sample /Users/yaseminbridges/full_test_pheval_run/corpora/corpus1/default/phenopackets/Abdul_Wahab-2016-GCDH-Patient_5.json --output-directory /Users/yaseminbridges/full_test_pheval_run/output/exomiser-13.2.0/default-corpus1-default/results --output-filename Abdul_Wahab-2016-GCDH-Patient_5-exomiser --preset phenotype-only`

Reply:
Why should this be fixed? Isnt this passed in as a command line parameter?

Reply:
Not within the PhEval run command. It's this path `/results/exomiser-13.2.0/runner_input_commands` -  during `pheval run` in the `run` step the batch files are prepared and output to this directory, and then exomiser/phen2gene runs using that batch file.

The naming for the 'batch' file needs to change to the same naming as {config}-{corpus}-{corpus-variant}-batch.txt, but this is the idea. Jules said it would be a good idea to keep the actual tool commands used to produce this output in case whoever is running it wants to run the jobs again but separate to the whole `pheval run`
```
├── exomiser-13.2.0
│   ├── default-corpus1-default
│   │   ├── pheval_gene_results
│   │   │   ├── Abdul_Wahab-2016-GCDH-Patient_5-pheval_gene_result.tsv
│   │   │   └── Ajmal-2013-BBS1-IV-5_family_A-pheval_gene_result.tsv
│   │   └── results
│   │       ├── Abdul_Wahab-2016-GCDH-Patient_5-exomiser.html
│   │       ├── Abdul_Wahab-2016-GCDH-Patient_5-exomiser.json
│   │       ├── Ajmal-2013-BBS1-IV-5_family_A-exomiser.html
│   │       └── Ajmal-2013-BBS1-IV-5_family_A-exomiser.json
│   └── runner_input_commands
│       └── default-exomiser-batch.txt
```

The directory `runner_input_commands` probably needs a better name so it can be separated from the `runner`, maybe `tool_input_commands`/`tool_commands` make more sense?

Reply:
renamed directory to `tool_input_commands` to reduce confusion with the runner